icechunk/src/config.rs

```diff
@@ -624,82 +624,35 @@ pub enum Credentials {
 #[allow(clippy::panic, clippy::unwrap_used, clippy::expect_used)]
 mod tests {
     use crate::{
-        ObjectStoreConfig, RepositoryConfig, config::S3Options,
-        strategies::repository_config, virtual_chunks::VirtualChunkContainer,
+        strategies::{repository_config,
```

Comment:
@ebarylko can you run `cargo fmt`? Otherwise the linter will complain and not allow merging